api4/user.go

```diff
@@ -1060,6 +1071,13 @@ func autocompleteUsers(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte((autocomplete.ToJson())))
 }
 
+func convertToUserPatch(user *model.User) *model.UserPatch {
```

Comment:
This should probably be a method on a user type and be moved to models/users.go.
func (u *User) IsUserOAuth(){}